srcpkgs/yq/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'yq'
 pkgname=yq
-version=2.11.1
-revision=2
+version=2.12.2
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 makedepends="python3-yaml"
-depends="python3-setuptools python3-xmltodict python3-yaml python3-toml python3-argcomplete"
-checkdepends="jq python3-xmltodict python3-toml python3-argcomplete"
+depends="${makedepends} python3-xmltodict python3-toml python3-argcomplete"
+checkdepends="${depends} jq"
```

Comment:
Shouldn't it also depend on `jq`, given that it's a wrapper around it?

Reply:
Yes it should